spyder/plugins/application/container.py

```diff
@@ -151,11 +162,41 @@ def setup(self):
             _("&Restart"),
             icon=self.create_icon('restart'),
             tip=_("Restart"),
-            triggered=self.sig_restart_requested,
+            triggered=self.restart_normal,
+            context=Qt.ApplicationShortcut,
+            shortcut_context="_",
+            register_shortcut=True)
+
+        self.restart_debug_action = self.create_action(
+            ApplicationActions.SpyderRestartDebug,
+            _("&Restart in debug mode"),
+            tip=_("Restart in debug mode"),
+            triggered=self.restart_debug,
             context=Qt.ApplicationShortcut,
             shortcut_context="_",
             register_shortcut=True)
 
+        # Debug logs
+        self.menu_debug_logs = self.create_menu(
+            LogsMenus.DebugLogsMenu, _("Debug logs"))
+        self.menu_debug_logs.aboutToShow.connect(self.update_debug_logs)
+
+    def update_debug_logs(self):
+        """Create an action for each lsp and debug log file."""
+        self.menu_debug_logs.clear()
+        debug_logs = []
+        files = glob.glob(os.path.join(get_conf_path('lsp_logs'), '*.log'))
+        files.append(os.environ['SPYDER_DEBUG_FILE'])
+        for f in files:
+            action = self.create_action(
+                f, os.path.basename(f), tip=f,
+                triggered=lambda: self.load_log_file(f), overwrite=True)
+            action.setData(f)
+            debug_logs.append(action)
+        add_actions(self.menu_debug_logs, debug_logs)
+        # for action in debug_logs:
+        #     self.add_item_to_menu(action, self.menu_debug_logs)
```

Comment:
@ccordoba12, `add_item_to_menu` does not populate the menu for some reason...but `add_actions` does.